### DISCUSSION TITLE
Make "final" results the default

### DISCUSSION THREAD

**Discussion by kerinin** [kaskada-ai/kaskada#227] [Ideas]
Historical results are consistently surprising to users, and are often encountered very early in the user journey. Many users don't read docs like a book - they just jump around to find examples and hack at them. Making final results the default would provide for a more incremental onboarding experience - users would receive the results they expect but still have the ability to get historical results when needed.

**Comment by bjchambers** [comment]
I like that it would be more "typical", but worry that in some ways this would hide the big benefit of Kaskada and timelines. I wonder if we had all examples specify explicitly either `--historic-results` or `--final-results` if that would help? Then if someone jumps around, they'd at least see an explicit option that they could look up and/or tweak.

I'm not sure I'd go as far as making it required though -- that seems like it would be "too much".
